core/types/json.go

```diff
@@ -44,6 +51,13 @@ func (b *hexBytes) UnmarshalJSON(input []byte) error {
 
 type hexBig big.Int
 
+func (b *hexBig) MarshalJSON() ([]byte, error) {
```

Comment:
We have decided to move all hex code to its own package. We'll take care of this issue then.